src/raftstore/store/fsm/peer.rs

```diff
@@ -315,20 +344,29 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
             CasualMessage::ClearRegionSize => {
                 self.on_clear_region_size();
             }
+            CasualMessage::RegionOverlapped => {
+                debug!("start ticking for overlapped"; "region_id" => self.region_id(), "peer_id" => self.fsm.peer_id());
+                // Maybe do some safe check first?
```

Comment:
Is this a TODO?

Reply:
It's an immature idea that I don't even sure if it's possible and necessary.